### PR TITLE
Update GrandNode Readme

### PR DESCRIPTION
**Pull request by pavlenex**
This PR improves readme.md and adds banner image.

To-do, once GrandNode plugin migrates to BTCPay Server Organization and is available to marketplace of GrandNode:

- [ ] Ensure links are correct
- [ ] Add link to GitHub download
- [ ] Add link to Marketplace download

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,71 @@
...
 
-Experience the simplicity of accepting Bitcoin payments with just a few straightforward steps. You can configure the plugin either automatically or manually, depending on your preferences and requirements.
+## Features
 
-## Automatic Configuration
+- 0% Fees: Enjoy a payment gateway with no fees. Yes, really!
+- No complicated KYC needed, you're your own, self-hosted payment gateway
+- Automated invoice management and refunds
+- Simple and customizable checkout UX, available in 30+ languages
+- Direct payments to your wallet, no intemediary
```
Comment (ndeet) [inline README.md]: typo, r missing -> intermediary

File: README.md [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
+
+## Getting Started 
+
+### Download and installation
+
+The plugins can be downloaded through GrandNode Extension Directory or GitHub. 
+
+For installing the plugin on GrandNode, you can either upload the plugin directly to the `/plugins` folder in your GrandNode main directory and restart your application, or use the Admin Panel to upload the plugin. 
+
+In the **Admin Panel**, navigate to **Plugins -> Local Plugins** and click the **"Upload"** button. Ensure the plugin ZIP file has the correct structure for successful upload. After uploading, find the plugin in the list, click "Install" next to it, and enable it if needed​.
```
Comment (ndeet) [inline README.md]: "Ensure the plugin ZIP file has the correct structure for successful upload." not sure if this bothers users, we need to make sure the structure is right when generating the zip?

File: README.md [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+## Plugin configuration
+
+You can configure the plugin either automatically or manually, depending on your preferences and requirements.
+
+### Automatic Configuration
 
 1. Enter Url to your BTCPay Server into "BTCPay Url" field. (e.g. https://mainnet.demo.btcpayserver.org)
-2. Click on the "Configure automatically" button to be redirected to the API authorization page on your BTCPay server
-3. On BTCPay authorization page: Select the store you want to connect to your GrandNode (you might need to login first)
+2. Click on the "Configure automatically" button to be redirected to the API authorization page of your BTCPay server
```
Comment (ndeet) [inline README.md]: capital S on BTCPay Server

File: README.md [hunk 4]
```diff
@@ -1,42 +1,71 @@
...
+3. On the authorization page: Select the store you want to connect to your GrandNode (you might need to login first)
 4. Click on "Authorize App" button and you will be redirected back to your GrandNode
 3. The "API Key", "BTCPay Store ID" and "Webhook Secret" fields will be automatically filled and a webhook created
 4. Click "Save" button at the top right to persist the configuration
```
Comment (ndeet) [inline README.md]: Button is at the bottom afaics

**Review by ndeet — CHANGES_REQUESTED** [review]
minor typos and nitpicks :)

File: README.md (hunk 4 above)
Comment (pavlenex) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pavlenex) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (pavlenex) [reply README.md]: Done

**Comment by pavlenex** [comment]
Implemented suggestions @ndeet, let me know if there's something else, asides from to-do I posted in OP.

**Review by ndeet — APPROVED** [review]

**Pull request merged**